metadata-ingestion/developing.md

```diff
@@ -13,24 +13,16 @@ The architecture of this metadata ingestion framework is heavily inspired by [Ap
 ### Requirements
 
 1. Python 3.6+ must be installed in your host environment.
-2. You also need to build the `mxe-schemas` module as below.
-   ```
-   (cd .. && ./gradlew :metadata-events:mxe-schemas:build)
-   ```
-   This is needed to generate `MetadataChangeEvent.avsc` which is the schema for the `MetadataChangeEvent_v4` Kafka topic.
-3. On MacOS: `brew install librdkafka`
-4. On Debian/Ubuntu: `sudo apt install librdkafka-dev python3-dev python3-venv`
-5. On Fedora (if using LDAP source integration): `sudo yum install openldap-devel`
+2. On MacOS: `brew install librdkafka`
+3. On Debian/Ubuntu: `sudo apt install librdkafka-dev python3-dev python3-venv`
+4. On Fedora (if using LDAP source integration): `sudo yum install openldap-devel`
 
 ### Set up your Python environment
 
 ```shell
-python3 -m venv venv
+../gradlew build -x check # setup venv and install
```

Comment:
Can this be a specific gradle task? 

such as: `../gradlew :metadata-ingestion:installPackage`

Reply:
yep I'll change that